unit2-java/intellij.rst

```diff
@@ -0,0 +1,30 @@
+Unit 2 (Java) - IntelliJ Learning Objectives
+=============================================
+
+Goals
+-----
+
+- Understand some of the tools and integrations that IntelliJ provides
+- Use IntelliJ to improve work efficiency and quality 
+
+Objectives
+----------
+
+Debugging
+^^^^^^^^^
+
+- Create breakpoints in source code
+- Run programs in debug mode
+- Inspect variables and parameters while in debug mode
+- Create conditional breakpoints
+
+IDE Features
```

Comment:
How about adding a bullet point or two about hovering over red lines/red words to pick up suggestions for fixing code errors?